### DISCUSSION TITLE
New GOTO Paths

### DISCUSSION THREAD

**Discussion by tznind** [HicServices/RDMP#1305] [General]
Post your new desired GOTO paths here.  Use the format From=>To.  For example:

`SelectedDataSets`=>`TableInfo`

If you are unsure what an object is right click it and select `What is this?`

**Comment by zzszarka** [comment]
From Extraction window => View Logs
Burger button and right click on tab.
![image](https://user-images.githubusercontent.com/77501138/180960540-ddd9e25f-67b9-4c6b-bd8e-3f869b69ba24.png)
